inventory/sample.libvirt.example.com.d/inventory/group_vars/all.yml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
I see that we're using a number of individual hostvars instead of having what the cluster "looks" like inside of `all.yml`. This looks like it works -- but seems very different from the other infra setups (aws, gcp, etc.). What I'd prefer to see is something that looks similar to:

```
libvirt_infrastructure:
   libvirt_iso: /path/to/iso
   libvirt_http_host: address of http host
   libvirt_os_variant: rhel7
   masters:
     count: 1
     cpu: 1
     memory: 2048
     name_prefix: <REPLACE WITH DESIRED NAME PREFIX FOR THE INSTANCE>
     root_volume_size: 15
     ...
```

We could also look at using a "cluster template" similar to how AWS relies on a cloud formation template and GCP relies on a deployment template. I believe there's an equivalent Libvirt component to this, so perhaps a combination of the above and this could be an option? 

Interested in seeing others thoughts here as well. I'd like to make sure we have a solid foundation here as I'm sure making changes here will require changes to the other steps in these PR's.

cc/ @oybed